doc/arkode/guide/source/Usage/User_callable.rst

```diff
@@ -3470,6 +3471,40 @@ Retrieve a pointer for user data                       :c:func:`ARKodeGetUserDat
    .. versionadded:: 6.1.0
 
 
+.. c:function:: int ARKodeGetNumRhsEvals(void* arkode_mem, int partition_index, long int* num_rhs_evals)
+
+   Returns the number of calls to the user's right-hand side function (so far).
+   For implicit methods or methods with an implicit partition, the count does
+   not include calls made by a linear solver or preconditioner.
+
+   :param arkode_mem: pointer to the ARKODE memory block.
+   :param num_partition: the right-hand side partition index:
+
+                   * For ERKStep, ``0`` corresponds to :math:`f(t,y)`
+
+                   * For ARKStep, ``0`` corresponds to :math:`f^E(t,y)` and
+                     ``1`` to :math:`f^I(t,y)`
+
+                   * For MRIStep, ``0`` corresponds to :math:`f^E(t,y)` and
+                     ``1`` to :math:`f^I(t,y)`
+
+                   * For SPRKStep, ``0`` corresponds to :math:`f_1(t,p)` and
+                     ``1`` to :math:`f_2(t,q)`
+
+                   For methods with more than one right-hand side function, a
```

Comment:
```suggestion
                   A
```
I think it's ok to document that a negative value works fine for ERKStep or anything with 1 partition.